packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -26013,396 +26106,197 @@ Map {
         "type": "bool",
       },
-      "iconDescription": Object {
-        "type": "string",
-      },
-      "kind": Object {
-        "args": Array [
-          Array [
-            "error",
-            "info",
-            "info-square",
-            "success",
-            "warning",
-            "warning-alt",
-          ],
-        ],
-        "isRequired": true,
-        "type": "oneOf",
+      "native": Object {
+        "type": "bool",
       },
-      "lowContrast": Object {
+      "nested": Object {
         "type": "bool",
       },
-      "notificationType": Object {
+    },
+  },
+  "OverflowMenu" => Object {
+    "$$typeof": Symbol(react.forward_ref),
+    "render": [Function],
+  },
+  "OverflowMenuItem" => Object {
+    "defaultProps": Object {
+      "disabled": false,
+      "hasDivider": false,
+      "isDelete": false,
+      "itemText": "Provide itemText",
+      "onClick": [Function],
+      "onKeyDown": [Function],
+    },
+    "propTypes": Object {
+      "className": Object {
         "type": "string",
       },
-      "onCloseButtonClick": Object {
+      "closeMenu": Object {
+        "type": "func",
+      },
+      "disabled": Object {
+        "type": "bool",
+      },
+      "handleOverflowMenuItemFocus": Object {
         "type": "func",
       },
-      "role": Object {
-        "isRequired": true,
-        "type": "string",
+      "hasDivider": Object {
+        "type": "bool",
       },
-      "statusIconDescription": Object {
+      "href": Object {
         "type": "string",
       },
-      "subtitle": Object {
+      "index": Object {
+        "type": "number",
+      },
+      "isDelete": Object {
+        "type": "bool",
+      },
+      "itemText": Object {
+        "isRequired": true,
         "type": "node",
       },
-      "timeout": Object {
-        "type": "number",
+      "onBlur": Object {
+        "type": "func",
+      },
+      "onClick": Object {
+        "type": "func",
+      },
+      "onFocus": Object {
+        "type": "func",
+      },
+      "onKeyDown": Object {
+        "type": "func",
+      },
+      "onKeyUp": Object {
+        "type": "func",
+      },
+      "onMouseDown": Object {
+        "type": "func",
+      },
+      "onMouseEnter": Object {
+        "type": "func",
+      },
+      "onMouseLeave": Object {
+        "type": "func",
+      },
+      "onMouseUp": Object {
+        "type": "func",
+      },
+      "primaryFocus": [Function],
+      "requireTitle": Object {
+        "type": "bool",
       },
       "title": Object {
-        "isRequired": true,
+        "type": "string",
+      },
+      "wrapperClassName": Object {
         "type": "string",
       },
     },
   },
-  "InlineNotification" => Object {
+  "Pagination" => Object {
     "defaultProps": Object {
-      "hideCloseButton": false,
-      "iconDescription": "closes notification",
-      "notificationType": "inline",
-      "onCloseButtonClick": [Function],
-      "role": "alert",
```

Comment:
The loss of all these defaultProps and propTypes seems to be the result of Carbon's new createComponentToggle method used to roll-out v11 updates via feature-flags. https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/Notification/index.js#L46